### DISCUSSION TITLE
Add support for home-assistant-viomi-vacuum-v8

### DISCUSSION THREAD

**Discussion by jsantosdias** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#386] [General]
Please, add support for https://github.com/tykarol/home-assistant-viomi-vacuum-v8

Thank you!

**Comment by PiotrMachowski** [comment]
Please create an issue using [this template](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/new?assignees=PiotrMachowski&labels=new+platform&template=new_platform_request.yml)
